site/cypress/e2e/Cypress-Feature/polls.spec.js

```diff
@@ -215,15 +215,15 @@ describe('Test cases revolving around polls functionality', () => {
         cy.contains('Add Response').click();
         cy.contains('Add Response').click();
         cy.contains('Add Response').click();
-        cy.get('#response_0_wrapper').children(':nth-child(3)').check();
-        cy.get('#response_0_wrapper').children(':nth-child(4)').type('Answer 1');
-        cy.get('#response_1_wrapper').children(':nth-child(4)').type('Answer 2');
-        cy.get('#response_2_wrapper').children(':nth-child(3)').check();
-        cy.get('#response_2_wrapper').children(':nth-child(4)').type('Answer 3');
+        cy.get('[data-testid="response_0_wrapper"]').children(':nth-child(3)').check();
```

Comment:
we are consistent with `hyphens(-)` instead of underscore `_` in data testid attributes

Reply:
[83ed327](https://github.com/Submitty/Submitty/pull/10197/commits/83ed327bb4406b4257bd2e013b8b883038ca9b5f)